### PR TITLE
MAINTAINERS: collect GPG fingerprints

### PR DESCRIPTION
**Pull request by AkihiroSuda**
The Committers who make releases SHOULD sign tags and release artifacts with GPG.
( I'm going to make this "MUST" for nerdctl: https://github.com/containerd/nerdctl/pull/2089 )

### CODE REVIEW

**Review by dmcgowan — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

File: MAINTAINERS [hunk 1]
```diff
@@ -3,37 +3,37 @@
...
+"dchen1107","Dawn Chen","dawnchen@google.com",""
+"Random-Liu","Lantao Liu","lantaol@google.com",""
+"mikebrow","Mike Brown","brownwm@us.ibm.com",""
+"yujuhong","Yu-Ju Hong","yjhong@google.com",""
+"fuweid","Fu Wei","fuweid89@gmail.com",""
+"mxpv","Maksym Pavlenko","pavlenko.maksym@gmail.com",""
+"dims","Davanum Srinivas","davanum@gmail.com",""
+"kevpar","Kevin Parsons","kevpar@microsoft.com",""
+"kzys","Kazuyoshi Kato","katokazu@amazon.com",""
+"samuelkarp","Samuel Karp","me@samuelkarp.com","910C 2860 8D33 DDE6 89C0 3290 997C 5A3C D316 7CB5"
```
Comment (samuelkarp) [inline MAINTAINERS:21]: I maintain two different keys separated by context: work and personal.  This key is the work key associated with an email address from my employer and present on an employer-controlled machine.  0A4B DF41 8E8D ECB8 7F3E  9E14 AAA3 FE8A 831F C087 is my personal key present only on personal infrastructure.

Can we list multiple keys here?  If there's only one, I'd prefer 0A4B DF41 8E8D ECB8 7F3E  9E14 AAA3 FE8A 831F C087.

File: MAINTAINERS [hunk 2]
```diff
@@ -3,37 +3,37 @@
...
+"dmcgowan","Derek McGowan","derek@mcgstyle.net","8C7A 111C 2110 5794 B0E8 A27B F58C 5D0A 4405 ACDB"
+"estesp","Phil Estes","estesp@gmail.com","71AE 6DCD DFF8 C2D1 DDCA EEC9 FE25 9812 6B19 6A38"
+"stevvooe","Stephen Day","stevvooe@gmail.com",""
+"dchen1107","Dawn Chen","dawnchen@google.com",""
+"Random-Liu","Lantao Liu","lantaol@google.com",""
+"mikebrow","Mike Brown","brownwm@us.ibm.com",""
+"yujuhong","Yu-Ju Hong","yjhong@google.com",""
+"fuweid","Fu Wei","fuweid89@gmail.com",""
+"mxpv","Maksym Pavlenko","pavlenko.maksym@gmail.com",""
+"dims","Davanum Srinivas","davanum@gmail.com",""
```
Comment (dims) [inline MAINTAINERS:18]: mine is `A67E 5FD8 80EB 089F 2317 7967 80D8 3A79 6103 BF59`

**Review by fahedouch — APPROVED** [review]

File: MAINTAINERS (hunk 2 above)
Comment (AkihiroSuda) [reply MAINTAINERS]: Added

File: MAINTAINERS (hunk 1 above)
Comment (AkihiroSuda) [reply MAINTAINERS]: Added two keys

File: MAINTAINERS [hunk 3]
```diff
@@ -3,37 +3,37 @@
...
+"dchen1107","Dawn Chen","dawnchen@google.com",""
+"Random-Liu","Lantao Liu","lantaol@google.com",""
+"mikebrow","Mike Brown","brownwm@us.ibm.com",""
+"yujuhong","Yu-Ju Hong","yjhong@google.com",""
+"fuweid","Fu Wei","fuweid89@gmail.com",""
+"mxpv","Maksym Pavlenko","pavlenko.maksym@gmail.com",""
+"dims","Davanum Srinivas","davanum@gmail.com","A67E 5FD8 80EB 089F 2317 7967 80D8 3A79 6103 BF59"
+"kevpar","Kevin Parsons","kevpar@microsoft.com",""
+"kzys","Kazuyoshi Kato","katokazu@amazon.com",""
+"samuelkarp","Samuel Karp","me@samuelkarp.com","0A4B DF41 8E8D ECB8 7F3E 9E14 AAA3 FE8A 831F C087,910C 2860 8D33 DDE6 89C0 3290 997C 5A3C D316 7CB5"
```
Comment (AkihiroSuda) [inline MAINTAINERS:21]: @samuelkarp LGTY?

**Review by samuelkarp — APPROVED** [review]

**Pull request merged**